packages/web/docs/src/content/specs/link-specs.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: 'Hive Specs: Federated Features'
+---
+
+import NextImage from 'next/image'
+import { Tabs } from '@theguild/components'
+
+# GraphQL Hive Technical Specifications
+
+Federation `@link` compliant features for GraphQL Hive.
+
+---
+
+## Introduction
+
+Links are a feature in Federation 2.x that allow external specifications to be referenced and used
+by a subgraph. Hive uses links to allow schemas to opt in to schema related features, such as
+metadata.
+
+### Importing via `@link`
+
+A preqrequisite is that the subgraph is using Federation 2.x:
+
+```graphql
+extend schema
+  @link(url: "https://specs.apollo.dev/link/v1.0")
+  @link(url: "https://specs.apollo.dev/federation/v2.3")
+```
+
+Then the Hive features can be imported like:
+
+```graphql
+extend schema @link(url: "https://specs.graphql-hive.com/hive/v1.0", import: ["@meta"])
+```
+
+Be sure to import and add definitions for all resources you intend to use.
+
+### Subgraph schema additions
+
+```graphql
+directive @meta(
+  name: String!
+  content: String!
+) repeatable on SCHEMA | OBJECT | INTERFACE | FIELD_DEFINITION
+```
+
+## Features
+
+### @meta
+
+<Tabs items={["v1.0"]}>
+<Tabs.Tab>
+
+Use this directive to enhance schema types with additional data, visible via the Hive Console. _This
+intentionally does not appear in the API Schema to avoid bloat. But if you want this directive to be
+returned to your gateway, then you can use `@composeDirective`_
```

Comment:
Should there be an example of using `@composeDirective`?

Reply:
Good question. I could definitely provide one. Based on chatting with Kamil, I should maybe just not mention this in the docs. Or leave it at composeDirective being unnecessary.
The reason I mentioned it was in case teams want to piggyback off this directive to write other custom plugins in the gateway. But it may be too early to say if that would happen.